### PR TITLE
{Doc} Improve the main help

### PR DESCRIPTION
**Pull request by jiasli**
**Description**  
Improve the main help by adding related links.

**Testing Guide**  
```
> az

     /\
    /  \    _____   _ _  ___ _
   / /\ \  |_  / | | | \'__/ _\
  / ____ \  / /| |_| | | |  __/
 /_/    \_\/___|\__,_|_|  \___|


Welcome to the cool new Azure CLI!

Documentation: https://docs.microsoft.com/en-us/cli/azure
GitHub repo: https://github.com/Azure/azure-cli

Use `az --version` to display the current version in human-readable format, `az version` to display
the current version in JSON format.
Use `--help/-h` to display the help message for a command group or command, like `az vm --help`, `az
vm create -h`.

Here are the base command groups and commands:

    account             : Manage Azure subscription information.
    acr                 : Manage private registries with Azure Container Registries.
    ad                  : Manage Azure Active Directory Graph entities needed for Role Based Access
    ...
```

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S172

File: src/azure-cli-core/azure/cli/core/_help.py [hunk 1]
```diff
@@ -44,8 +44,13 @@
 
 Welcome to the cool new Azure CLI!
 
-Use `az --version` to display the current version.
-Here are the base commands:
+Documentation: https://docs.microsoft.com/en-us/cli/azure
+GitHub repo: https://github.com/Azure/azure-cli
+
+Use `az --version` to display the current version in human-readable format, `az version` to display the current version in JSON format.
```
Comment (yungezz) [inline src/azure-cli-core/azure/cli/core/_help.py:50]: is it necessary to put this detail in the main message?

File: src/azure-cli-core/azure/cli/core/_help.py [hunk 2]
```diff
@@ -44,8 +44,13 @@
 
 Welcome to the cool new Azure CLI!
 
-Use `az --version` to display the current version.
-Here are the base commands:
+Documentation: https://docs.microsoft.com/en-us/cli/azure
```
Comment (yungezz) [inline src/azure-cli-core/azure/cli/core/_help.py:47]: pls use short url aka.ms/cli

File: src/azure-cli-core/azure/cli/core/_help.py (hunk 1 above)
Comment (jiasli) [reply src/azure-cli-core/azure/cli/core/_help.py:50]: I am not sure. Maybe we can totally remove this line.

File: src/azure-cli-core/azure/cli/core/_help.py (hunk 2 above)
Comment (jiasli) [reply src/azure-cli-core/azure/cli/core/_help.py:47]: Short link is difficult to maintain, given the conditional that https://aka.ms can't use an email group as short link owner. Let's discuss in more detail about short links.

**Pull request closed without merging**